tests/bits/dof_tools_01a.cc

```diff
@@ -57,9 +57,7 @@ check_this (const DoFHandler<dim> &dof_handler)
   unsigned int hash = 0;
   for (unsigned int l=0; l<sp.n_rows(); ++l)
     hash += l*(sp.row_length(l) +
-               sp.get_rowstart_indices()[l] +
-               sp.get_column_numbers()[sp.get_rowstart_indices()[l]
-                                       +
-                                       (sp.row_length(l)>1 ? 1 : 0)]);
+               (sp.begin(l)-sp.begin()) +
+               (sp.row_length(l)>1 ? ++sp.begin(l)++ : sp.begin(l))->column());
```

Comment:
Is there any deeper meaning in the post-fix increment for sp.begin(l) that I don't see? You're interested in entry 1, so the prefix increment should be enough.